airdrop/airdrop-distributor/tests/functions/constructor.rs

```diff
@@ -0,0 +1,82 @@
+use crate::utils::{
+    airdrop_distributor_abi_calls::{airdrop_constructor, end_block, merkle_root},
+    test_helpers::setup,
+};
+
+mod success {
+
+    use super::*;
+
+    #[tokio::test]
+    async fn initalizes() {
+        let (deploy_wallet, _, _, _, asset) = setup().await;
+
+        assert_eq!(end_block(&deploy_wallet.airdrop_distributor).await, 0);
+        assert_eq!(
+            merkle_root(&deploy_wallet.airdrop_distributor).await,
+            [0u8; 32]
+        );
+
+        airdrop_constructor(
+            10,
+            &deploy_wallet.airdrop_distributor,
+            [1u8; 32],
+            asset.asset_id,
+        )
+        .await;
+
+        // TODO: Get block height and add 10
```

Comment:
[Block manipulation](https://fuellabs.github.io/fuels-rs/v0.22.0/testing/chains.html) is now a thing, could you try it?